app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
I always felt like `multi-bid` and `single-bid` clarified this stuff pretty well in my mind :/

But I guess both of those are sub-types of Reverse. And spit-balling a bit more here, other plausible auction types:

- `Dutch` (probably not a child of `Reverse`) https://en.wikipedia.org/wiki/Dutch_auction
- `BuyItNow` or `FixedPrice` (should be composable with with any Reverse auction. e.g. something can be BuyItNow and MultiBid.
- `Lot` or `MultiWinner`: There can be many winners, maybe used for a crowdsourcing micro-purchase. Should likely inherit from `ButItNow` (assuming these would just have a fixed price).